packages/rules-unit-testing/test/database.test.ts

```diff
@@ -42,18 +42,29 @@ describe('Testing Module Tests', function () {
       .catch(() => {});
   });
 
-  it('assertFails() iff failure', async function () {
+  it('assertFails() iff PERMISSION_DENIED', async function () {
```

Comment:
@yuchenshi do you have a suggestion for what a meaningful test of `initializeAdminApp` would look like?  The current one checks the internal token provider for `owner` but that's no longer relevant.

Reply:
We should just call one of the admin-only operations (list collections?) and assert that succeeds.